README.md

```diff
@@ -98,14 +98,11 @@ before diving in here. That will introduce you to the various components used.
     > everytime your session expires.
  
 4. Technically you're now able to use the aws cli to perform any commands through Airlock
-   to S3. Airlock automatically creates the user on Ceph for you. One thing it cannot do though, is make this user an 
-   admin/system user. Because of this, users on Ceph are not allowed to perform actions on other users' buckets.
+   to S3. Airlock automatically creates the user on Ceph for you. Since Ranger is in place to handle authorisation,
+   all users on Ceph can be allowed to do everything.
```

Comment:
I would say maybe that:
all user authorization is handled on ranger level, so required permissions can be defined for users.
everything sounds not right from sec point of view

Reply:
That sentence was already in the README (line 104).

Users in Ceph are indeed able to do everything (they all are admin), because authorization is provided by Ranger. I will rephrase it to make it clearer.